### DISCUSSION TITLE
API calls with Cypress

### DISCUSSION THREAD

**Discussion by punisher49** [cypress-io/cypress#16155] [Questions and Help] [answered]
I have a scenarios where I need to make first API call and then copy generated ID from first API, then pass it to the second API call, then copy generated link from second API and only after that start my Cypress tests. 
I wondering if it is possible with Cypress and worth my time to spend on it?

**Comment by bahmutov** [comment, marked as answer]
You can execute these calls in the plugin file. You can find similar examples in the "Dynamic ..." recipes in https://github.com/cypress-io/cypress-example-recipes#fundamentals

I probably should add a recipe showing exactly how to do so, if interested, could you open an issue in that repo?
